client/src/pages/login/index.tsx

```diff
@@ -40,6 +40,10 @@ function LoginPage() {
     if (checkIsLoggedIn()) {
       goToHome();
     }
+
+    if (checkIsExpired()) {
+      setError('Your session has expired. Please login again to continue.');
```

Comment:
Are we removing the expired token from storage as well?

Reply:
Yea let me add that in, makes sense since we prompt the message